db-client-java/src/main/java/com/eventstore/dbclient/ConnectionBuilder.java

```diff
@@ -12,6 +12,7 @@
 public class ConnectionBuilder {
     private Timeouts _timeouts;
     private SslContext _sslContext = null;
+    private boolean _keepAlive = false;
```

Comment:
**QUESTION:** I'm not sure if that was the conclusion but didn't we say that we should have `keepAlive` set as true by default?

Reply:
Shouldn't we also allow the user to set `keepalive_timeout_ms`?